.circleci/config.yml

```diff
@@ -291,21 +264,10 @@ jobs:
 
   build-ios-release:
     executor: ios
-    environment:
-      APP_NAME: "Mattermost"
-      REPLACE_ASSETS: true
-      MAIN_APP_IDENTIFIER: "com.mattermost.rn"
-      SENTRY_ENABLED: true
-      SEGMENT_API_KEY: ${SEGMENT_API_KEY_PROD}
-      MATCH_TYPE: appstore
-      MATCH_APP_IDENTIFIER: 'com.mattermost.rn.NotificationService,com.mattermost.rn.MattermostShare,com.mattermost.rn'
-      EXTENSION_APP_IDENTIFIER: 'com.mattermost.rn.MattermostShare'
-      NOTIFICATION_SERVICE_IDENTIFIER: 'com.mattermost.rn.NotificationService'
-      IOS_APP_GROUP: 'group.com.mattermost'
-      IOS_ICLOUD_CONTAINER: 'iCloud.com.mattermost.rn'
-      IOS_BUILD_EXPORT_METHOD: 'app-store'
-      AWS_FOLDER_NAME: 'mattermost-mobile'
-      MATTERMOST_WEBHOOK_URL: ${MATTERMOST_WEBHOOK_BUILD}
+    environment: # remove env vars on final PR
+      SENTRY_ENABLED: false
+      AWS_FOLDER_NAME: ''
+      MATTERMOST_WEBHOOK_URL: ''
```

Comment:
is it intended to have different values from the previous change?

Reply:
☝️